src/main/common/huffman.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <stdint.h>
+
+#define HUFFMAN_EOF (-1)
+
+#define HUFFMAN_TABLE_SIZE 257 // 256 characters plus EOF
+typedef struct huffmanTable_s {
```

Comment:
Well, I've only tried a few black box logs, but I've already seen ratios of 1200:1 (there are an awful lot of zeros in BB logs). So I wouldn't discount the possibility of ratios in excess of 4096:1. I need to get hold of a variety of logs from different sources.